src/Packetery/Core/Validator/Order.php

```diff
@@ -64,6 +82,68 @@ public function validate( Entity\Order $order ): bool {
 		);
 	}
 
+	/**
+	 * Validates data needed to submit packet and returns array of errors.
+	 *
+	 * @param Entity\Order $order Order entity.
+	 *
+	 * @return string[]
+	 * @throws TranslationMissingException For the case required translation is not set.
+	 */
+	public function getValidationErrors( Entity\Order $order ): array {
+		$errors = [];
+		if ( ! $order->getNumber() ) {
+			if ( empty( $this->translations[ self::ERROR_TRANSLATION_KEY_NUMBER ] ) ) {
+				throw new TranslationMissingException( 'Order validator must have all translations set' );
```

Comment:
Todle bych nedělal. Pokud bude PHP tahat neexistující klíč z translations tak ti to Tracy zobrazí v liště. To si myslím stačí. A eshopista se do tohoto stavu s těmito změnami nikdy nedostane, pokud to máš teď dobře.

Reply:
Chybějící překlad, to může snadno někomu v budoucnosti stát a nechci spoléhat na to, že se k technické podpoře dostane "undefined array key". V případě PacketaPickupPointsConfigu také máme assert.

Reply:
Tak tu extrakci překladu zabal do privátní metody a do ní dej ten if ať se to tam neopakuje.